src/test/scala/org/ergoplatform/nodeView/history/VerifyADHistorySpecification.scala

```diff
@@ -91,81 +82,6 @@ class VerifyADHistorySpecification extends HistoryTestHelpers with NoShrink {
     fullBlocksToApply.head.blockSections.foreach(s => history.applicable(s) shouldBe false)
   }
 
-  property("proofs and transactions application in random order with forks") {
-    forAll(smallInt, positiveLongGen) { (chainHeight, seed) =>
-      whenever(chainHeight > 0) {
-        val (history, chain) = genHistory(1)
-        val r = new Random(seed)
-        val genesis = chain.head
-        history.bestFullBlockOpt shouldBe Some(genesis)
-
-        val chains = Seq(genChain(chainHeight, genesis), genChain(chainHeight + 1, genesis)).map(_.tail)
-        chains.foreach(chain => applyHeaderChain(history, HeaderChain(chain.map(_.header))))
-        val indices: Seq[(Int, Int)] = chains.indices.flatMap { chainIndex =>
-          val chain = chains(chainIndex)
-          chain.indices.map(blockIndex => (chainIndex, blockIndex))
-        }
-
-        var appended: ArrayBuffer[ErgoFullBlock] = ArrayBuffer.empty
-
-        def findBestBlock(appendedToCheck: Seq[ErgoFullBlock]): ErgoFullBlock = {
-          def firstInAppended(h: Header): Header = {
-            appended.find(_.header.id == h.parentId).map(_.header) match {
-              case Some(prev) => firstInAppended(prev)
-              case None => h
-            }
-          }
-
-          if (appendedToCheck.isEmpty) {
-            genesis
-          } else {
-            val best = appendedToCheck.maxBy(_.header.height)
-            if (firstInAppended(best.header).parentId == genesis.id) {
-              best
-            } else {
-              findBestBlock(appendedToCheck.filterNot(_.id == best.id))
-            }
-          }
-        }
-
-        r.shuffle(indices).foreach { i =>
-          val block = chains(i._1)(i._2)
-          r.shuffle(block.blockSections).foreach(s => history.append(s) shouldBe 'success)
-
-          appended += block
-
-          findBestBlock(appended).header.height shouldBe history.bestFullBlockOpt.get.header.height
-        }
-      }
-    }
-  }
-
-  property("apply proofs that link incomplete chain") {
```

Comment:
why this test is removed?

Reply:
@catena2w, As soon as blocks could be applied only sequentially there could not be incomplete chain, so this test does not make sense.